package.json

```
@@ -22,18 +22,17 @@
     "@tootallnate/once": "2.0.0",
     "async-listen": "1.2.0",
     "debug": "4.3.4",
-    "execa": "3.2.0",
-    "fs-extra": "8.1.0",
     "generic-pool": "3.4.2",
     "micro": "9.3.5-canary.3",
     "ms": "2.1.1",
-    "node-fetch": "2.6.7",
+    "node-fetch": "^2.6.7",
```

```suggestion
    "node-fetch": "2.6.7",
```
Prefer pinning version.